### PR TITLE
Add the VMCAI 2020 paper

### PR DESCRIPTION
**Pull request by shaobo-he**

### CODE REVIEW

File: _publications/2020_vmcai_gbhr.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+# The shortname is used for auto-generated titles
+shortname: VMCAI 2020
+# add a 2:1 aspect ratio (e.g., width: 400px, height: 200px) to the folder /assets/images/papers/
+image: 2020_vmcai_gbhr.png
+# add a 2:1 aspect ratio teaser figure (e.g., width: 1200px, height: 600px) to the folder /assets/images/papers/
+image_large: 2020_vmcai_gbhr_teaser.png
+
+# Authors in the "database" can be used with just the key (lastname). Others can be written properly.
+authors:
+- Jack J. Garzella
```
Comment (zvonimir) [inline _publications/2020_vmcai_gbhr.md]: This should be just `garzella`.

File: _publications/2020_vmcai_gbhr.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+# The reference to the preview viedo entry
+#preview-video:
+
+# the preprint
+pdf: 2020_vmcai_gbhr.pdf
+# A supplement PDF
+supplement: 
+
+# Extra supplements, such as talk slides, data sets, etc.
+supplements:
```
Comment (zvonimir) [inline _publications/2020_vmcai_gbhr.md:65]: It would be good to add a link to our benchmark suite into supplements.

**Review by zvonimir — CHANGES_REQUESTED** [review]

File: _publications/2020_vmcai_gbhr.md (hunk 1 above)
Comment (shaobo-he) [reply _publications/2020_vmcai_gbhr.md]: Done.

File: _publications/2020_vmcai_gbhr.md (hunk 2 above)
Comment (shaobo-he) [reply _publications/2020_vmcai_gbhr.md:65]: I put the github link in but I couldn't find it when tested it on my local machine.

Comment (zvonimir) [reply _publications/2020_vmcai_gbhr.md:65]: See this file for how to properly deal with supplements: https://github.com/visdesignlab/visdesignlab.github.io/blob/master/_publications/2019_eurovis_implicit-discretization.md

**Review by zvonimir — APPROVED** [review]
Looks good to me.

**Pull request merged**